platform-includes/crons/setup/php.laravel.mdx

```diff
@@ -2,7 +2,7 @@
 
 Use the Laravel SDK to monitor and notify you if your [scheduled task](https://laravel.com/docs/10.x/scheduling) is missed (or doesn't start when expected), if it fails due to a problem in the runtime (such as an error), or if it fails by exceeding its maximum runtime.
 
-To set up, add the `sentryMonitor()` macro to your scheduled tasks defined in your `app/Console/Kernel.php` file:
+To set up, add the `sentryMonitor()` macro to your scheduled tasks defined in your `app/Console/Kernel.php` file (please note that this will create a Cron monitor that will count against your quota):
```

Comment:
```suggestion
To set up, add the `sentryMonitor()` macro to the scheduled tasks defined in your `app/Console/Kernel.php` file as shown below. (Please note, this will create a Cron monitor that will count against your quota.)
```